docs/Plugins.md

```diff
@@ -185,11 +185,59 @@ The matching simplestream config to send audio from talkgroup 58918 to TCP port
         }
 ```
 
+## Community Plugins
+Community plugins can extend the features of Trunk Recorder and allow customized workflows or analysis.  
+> As new plugins are developed, authors are encouraged to add to the below tables by submitting a PR to this document.
+
+#### External Plugins
+Plugins that are built out-of-tree and installed seperately from Trunk Recorder:
```

Comment:
Are these technically "plugins"? Seems like they're just external applications that consume output.

Reply:
Yes.  There are a few examples such at my [MQTT fork](https://github.com/taclane/trunk-recorder-mqtt-status), that have a separate repository and are built against the headers and libraries installed with trunk-recorder.  Such plugins are functionally the same as the "built-in" plugins, and are loaded as libraries in the config.json.

I believe this was the intended method for plugin development, as it leaves the source entirely tree alone.  But it's got it's own set of issues, most notably falling out-of-sync with any trunk-recorder updates, and it forces the user to find and recompile any plugins whenever there are any changes to main libraries.